technical/20180530_hpm5g-radio-tests.md

```diff
@@ -0,0 +1,183 @@
+---
+title: HPM5G Radio Tests
+location: Toronto
+attendees: 8
+date: 2018-05-30
+---
+
+# Overview 
+
+This document summarizes a series of tests on the **Dragino HPM5G mPCIe radio module** and the **LibreRouter's 12 dBi antenna** that are expected to ship with the soon-to-be-released LibreRouter bundle. Toronto Mesh acquired six radio boards and antennas from Dragino, and in this test, used a pair of the [Marvell ESPRESSObin](http://espressobin.net) running [prototype](https://github.com/tomeshnet/prototype-cjdns-pi) to test the radios.
+
+![Dragino HPM5G mPCIe radio module](../images/20180530_hpm5g-radio-tests.jpg?raw=true)
+![LibreRouter's 12 dBi antenna](../images/20180530_hpm5g-radio-tests2.jpg?raw=true)
+
+## Issues
+
+Using our ESPRESSObin setup, the HPM5G board was automatically identified and brought up on boot with the `ath9k` driver. Throughout testing, we observed a couple issues that we were able to work around.
+
+### Power Issues on the ESPRESSObin mPCIe
+
+During the initial setup and testing we noticed that the ESPRESSObin would restart at what seemed to be random times. We tried different combination of radio boards and ESPRESSObin boards but the issue was persistent. We tried to reach out to the ESPRESSObin community without any luck; however the Armbian community mentioned that this seemed like an issue with insufficient power. Identifying this as a potential power issue we tried using larger amperage power supplies (1.5 and 2 A at 12 V) however this did not seem to solve the problem.
+
+**Work Around**
+
+After identifying a potential power problem we tried to limit the amount of power the radio would need by reducing the TX power of the radio. We found that keeping it around 14 dBm would let the ESPRESSObin operate without reboot. Our initial testing was then done using a 14 dBm TX power.
+
+We did run into an issue where the TX power would not be persistent after being set, due to the Network Manager Service changing its value. The solution was to change the regulatory database to prevent TX power higher then 14 dBm. This held the TX power at 14 dBm.
+
+**Long-term Solution**
+
+This issue is specific to the ESPRESSObin, since the LibreRouter mPCIe interfaces are designed to meet the power requirements of the HPM5G at 19 dBm. In later parts of this document, we will also describe a board modification that will allow us to power the HPM5G properly on the ESPRESSObin.
```

Comment:
No necessary exclusive to the ESPRESSObin, it may be an issue in other boards as well we do not know.  

Additionally power modification may allow the board to work with higher TX Power if powered externally on the LibreRouter as well.